### PR TITLE
Use portable shebang

### PR DESCRIPTION
**Pull request by rverchere**
For users using different python version, and using Mac OS X.

### CODE REVIEW

**Comment by peckpeck** [comment]
Thank you

File: rtf [hunk 1]
```diff
@@ -236,6 +236,12 @@ if Vagrant.has_plugin?("vagrant-cachier")
   config.cache.scope = :box
 end
 
+# Mac OS X workwaround
+config.vm.provision "fix-no-tty", type: "shell" do |s|
+    s.privileged = false
+    s.inline = "sudo sed -i '/tty/!s/mesg n/tty -s \\\\&\\\\& mesg n/' /root/.profile"
```
Comment (peckpeck) [inline rtf]: This is already done in scripts/cleanbox.sh isn't it working for you ?

Comment (rverchere) [inline rtf]: I guess no:

```
$ ./rtf platform setup debian
[...]
==> debian_server: Machine booted and ready!
==> debian_server: Checking for guest additions in VM...
==> debian_server: Setting hostname...
==> debian_server: Configuring and enabling network interfaces...
==> debian_server: Mounting shared folders...
    debian_server: /vagrant => /Users/rverchere/Workspace/automation/rudder/rudder-tests-official
==> debian_server: Running provisioner: shell...
    debian_server: Running: inline script
==> debian_server: stdin: is not a tty
==> debian_server: cannot open file fr
==> debian_server: apt-get is /usr/bin/apt-get
[...]
```

**Comment by peckpeck** [comment]
We usually have a ticket for each pull request. I opened one : http://www.rudder-project.org/redmine/issues/10353
Could you rebase your pull request into a single commit that starts with "Fixes #10353 "

Thank you !

**Comment by rverchere** [comment]
Done!

**Pull request merged**